### PR TITLE
Feature/134676703/static page management

### PR DESCRIPTION
**Pull request by oredavids**
**Are you submitting to `staging`?**

We currently only accept PR's to `staging` not `master`

#### What does this PR do?
Create and edit static pages, such as Contact, About, or Support.

#### Description of Task to be completed?
An admin/vendor can create and manage static pages with the help of WYSIWYG (What you see is what you get) editor.

#### How should this be manually tested?
Go to Pages Section on Dasbhoard to Create & Manage your pages

#### Any background context you want to provide?
NA
#### What are the relevant pivotal tracker stories?
Static Page Management #134676703

#### Questions:
NA

### CODE REVIEW

File: imports/plugins/core/layout/client/templates/layout/staticpages/shopView.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <hr>
+  <br>
+  {{ #each page in shopPage }}
+    <div class="shopPage">
+      <div>
+        <h2> {{ page.pageTitle }} </h2>
+      </div>
+      <hr>
+      <div>
+        <h4> {{{ page.pageBody }}} </h4>
```
Comment (azemoh) [inline imports/plugins/core/layout/client/templates/layout/staticpages/shopView.html:12]: Does this work or is it a typo? - Having 3 curly braces `{{{}}}`

Comment (oredavids) [reply imports/plugins/core/layout/client/templates/layout/staticpages/shopView.html:12]: This works. Its the syntax to ensure the page is rendered in user friendly text-format. The data contains html tags.

File: imports/plugins/core/layout/client/templates/layout/staticpages/shopView.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { StaticPages } from "/lib/collections";
+import { Template } from "meteor/templating";
+// import { Meteor } from "meteor/meteor";
```
Comment (kehindeoni-zz) [inline imports/plugins/core/layout/client/templates/layout/staticpages/shopView.js]: Remove commented code

File: imports/plugins/included/pages/client/templates/staticpage.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
+<template name="staticPage">
+
+  <div class="container">
+
```
Comment (kehindeoni-zz) [inline imports/plugins/included/pages/client/templates/staticpage.html]: Unnecessary line space, please remove line 4 and 5

**Pull request merged**